docs/user-guide/writing-your-docs.md

```diff
@@ -290,9 +290,9 @@ option.
 ```yml
 markdown_extensions:
     - toc:
-        permalink: "#"
-        baselevel: 2
-        separator: "_"
+          permalink: "#"
+          baselevel: 2
+          separator: "_"
```

Comment:
This doesn't appear to change the AST for YAML: https://astexplorer.net/#/gist/1a438c85ab91bef44fb8a41d8635b4e5/f5a7aa6724824304c7d43cbfc9f9eb4de985885c